api/v1/openapi.yaml

```diff
@@ -2731,6 +2731,9 @@ definitions:
       ipLocalReservedPorts:
         description: Comma-separated list of IP ports should be reserved in the workload network namespace
         type: string
+      CNIProxyRouting:
```

Comment:
Have you investigated if this could be part of the CNI config file instead? https://github.com/cilium/cilium/blob/f6606c6ccdb80c187243ec8359ddfb037aa75969/plugins/cilium-cni/types/types.go#L22

My understanding is that this file is available even in chaining mode. It is written here: https://github.com/cilium/cilium/blob/2311f3d7811c8950b7f5e1e2ac290f8216be0dc3/daemon/cmd/cni/config.go#L274

(Though I might be missing something here, I'm not intimately familiar with the CNI config lifecycle in chaining mode)